src/debugger.ts

```diff
@@ -46,7 +46,7 @@ function getFieldAndExecutable(config: vscode.DebugConfiguration): [keyof vscode
 function changeConfigForSip(config: vscode.DebugConfiguration, executableFieldName: string, executionInfo: MirrordExecution) {
```

Comment:
Also, can't this whole function be refactored to take care of all cases, such that it becomes an `if/if else/else`, sort of like

```ts
const command = config[executableFieldName];
if (config.type === "node-terminal" && command) {
  ...
} else if (executionInfo.patchedPath !== null) {
...
} else {
  throw new Exception(`SIP failed?`);
}
```